src/bundlers/bundler.interface.ts

```diff
@@ -21,6 +21,7 @@ export type BundlerInput = {
         commandParameters?: string[];
         installDeps?: boolean;
         allNonJsFilesPaths?: FileDetails[];
+        isOptimizationDisabled?: boolean;
```

Comment:
nit: I consider this a better naming.

```suggestion
        disableOptimization?: boolean;
```

Reply:
I didn't saw this comment in time and merged the PR. Still, I agree with the better naming, so I added it in subsequent commit. Thanks for the review!